Assets/Mirror/Weaver/Weaver.cs

```diff
@@ -179,7 +179,6 @@ class Weaver
         public static MethodReference sendEventInternal;
         #endregion
 
-        public static WeaverLists lists;
```

Comment:
Where did the declaration go?  it looks like WeaveList is not declared anywhere in this commit

Reply:
307612c

Reply:
I would have put it as part of the other commit,  but ok